shub/schedule.py

```diff
@@ -0,0 +1,41 @@
+from ConfigParser import NoSectionError, NoOptionError
+import click
+from click import ClickException
+from scrapinghub import Connection
+from shub import scrapycfg
```

Comment:
As we're shipping shub as a binary now, does it still make sense to be picky about having scrapy or not? Aren't we always bundling it already?